package.json

```diff
@@ -38,7 +38,6 @@
     "@fastify/helmet": "^11.1.1",
     "@fastify/rate-limit": "^9.1.0",
     "@fastify/websocket": "^10.0.1",
-    "@google-cloud/opentelemetry-cloud-monitoring-exporter": "^0.18.0",
```

Comment:
everything worked fine then?

Reply:
Yeah, this is just the Google specific one. We still have prometheus exporter